pkg/kubernetes/status.go

```diff
@@ -0,0 +1,98 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package kubernetes
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/dapr/cli/pkg/age"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+var (
+	controlPlaneLabels = []string{"dapr-operator", "dapr-sentry", "dapr-placement", "dapr-sidecar-injector"}
+)
+
+// StatutOutput represents the status of a named Dapr resource.
+type StatusOutput struct {
+	Name      string `csv:"NAME"`
+	Namespace string `csv:"NAMESPACE"`
+	Healthy   string `csv:"HEALTHY"`
+	Status    string `csv:"STATUS"`
+	Version   string `csv:"VERSION"`
+	Age       string `csv:"AGE"`
+	Created   string `csv:"CREATED"`
+}
+
+// List status for Dapr resources.
+func Status() ([]StatusOutput, error) {
+	client, err := Client()
+	if err != nil {
+		return nil, err
+	}
+
+	var wg sync.WaitGroup
+	wg.Add(len(controlPlaneLabels))
+
+	m := sync.Mutex{}
+	statuses := []StatusOutput{}
+
+	for _, lbl := range controlPlaneLabels {
+		go func(label string) {
+			p, err := client.CoreV1().Pods(v1.NamespaceAll).List(v1.ListOptions{
+				LabelSelector: labels.FormatLabels(
+					map[string]string{
+						"app": label,
```

Comment:
Could we please separate out in helper method?
p, err := getList(...)

Reply:
Done